### PR TITLE
No rainbow update documentation

### LINKED ISSUE #229: Update old documentation on easy quicklook display (stop using rainbow color map!)
#### Code Sample, a minimal, complete, and verifiable piece of code

```python
# Your code here

```
#### Problem description

The RTD pages on quick and easy display ("Displaying data quickly") uses rainbow color map and still uses Basemap. Since we are otherwise advocating against the rainbow color map, it is adequate to use another color map here. Also the example code is old (from the birth of Pyresample actually) and doesn't allow the user to replicate as the original input data is not there. Would be nice to use data from zenodo.

### PR DESCRIPTION
**Pull request by adybbroe**
- [x] Closes #229  
 - [x] Tests added 
 - [x] Tests passed 
 - [x] Passes ``git diff origin/master **/*py | flake8 --diff`` 
 - [x] Fully documented

### CODE REVIEW

File: docs/source/plot.rst [hunk 1]
```diff
@@ -23,47 +40,60 @@ of a dataset for a specified AreaDefinition. The function
...
   .. image:: _static/images/tb37v_quick.png
   
 The data passed to the functions is a 2D array matching the AreaDefinition.
 
 The Plate Carree projection
 +++++++++++++++++++++++++++
-The Plate Carree projection (regular lon/lat grid) is named **eqc** in Proj.4 and **cyl** in Basemap. pyresample uses the Proj.4 name.
-Assuming the file **areas.cfg** has the following area definition:
+The Plate Carree projection (regular lon/lat grid) is named **eqc** in
+Proj.4. Pyresample uses the Proj.4 nameing.
```
Comment (mraspaud) [inline docs/source/plot.rst]:
```suggestion
Proj.4. Pyresample uses the Proj.4 naming.
```

File: docs/source/plot.rst [hunk 2]
```diff
@@ -159,19 +208,69 @@ AreaDefinition using the **plot.area_def2basemap(area_def, **kwargs)** function.
...
+
+For more information on how to plot with Basemap please refer to the Basemap
+and matplotlib documentation.
+
+
+Adding background maps with Cartopy
+-----------------------------------
+
+As mentioned in the above warning Cartopy should be used rather than Basemap as
+the letter is not maintained anymore.
```
Comment (mraspaud) [inline docs/source/plot.rst]:
```suggestion
the latter is not maintained anymore.
```

**Comment by adybbroe** [comment]
Thanks for the finding the typos. I did a tiny bit of refactoring in the plotting part and added some more unittesting and hopefully increased the coverage a little bit

**Comment by adybbroe** [comment]
Okay, now that it pass on Travis I believe I am ready. I would argue the coverage that matters actually went up, but judge yourself if you bother? The one stickler complaint come from the fact that I wanted `_add_gridlines` to return the gridlines object, since I use it for unittesting. I assume this could be done in a different way thus skipping a return value...

**Pull request merged**